code/development-env/config/local/image_config.json

```diff
@@ -0,0 +1,97 @@
+{
+  "images": {
+    "RSHINY": {
+      "displayName": "Shiny",
+      "type": "site",
+      "userSelectableVersion": true,
+      "versions": [
+        {
+          "default": true,
+          "displayName": "4.0.1",
+          "image": "nerc/rshiny:4.0.1"
+        },
+        {
+          "displayName": "3.5.3",
+          "image": "nerc/rshiny:3.5.3"
+        }
+      ]
+    },
+    "RSTUDIO": {
+      "displayName": "RStudio",
+      "type": "notebook",
+      "userSelectableVersion": true,
+      "versions": [
+        {
+          "default": true,
+          "displayName": "4.0.1",
+          "image": "nerc/rstudio:4.0.1",
+          "connectVersion": "nerc/zeppelin-connect:1.1.1"
+        },
+        {
+          "default": false,
```

Comment:
The non-default version of `RSHINY` doesn't have this as `false`, it is just omitted. I would be inclined to go with omitting the default tag unless it is the default